src/Libraries/Microsoft.AspNetCore.HeaderParsing/HeaderParsingFeature.cs

```diff
@@ -198,12 +198,12 @@ private void CopyTo(Box<T> to)
         }
     }
 
-    [LoggerMessage(LogLevel.Debug, "Can't parse header '{headerName}' due to '{error}'.")]
+    [LoggerMessage(LogLevel.Debug, "Can't parse header '{HeaderName}' due to '{Error}'.")]
```

Comment:
I think we shouldn't apply a name [`http.request.header.<key>`](https://github.com/open-telemetry/semantic-conventions/blob/b127e121b9b458e83a0fee8a32f7eb6721912951/docs/attributes-registry/http.md) here, because we are not logging the value of the header here. The aim of this and other logger messages in this file is to log the name of the header, not its value, i.e. the data we should store in a log record should be `[HeaderName] = "name_of_the_header` rather than `["name_of_the_header"] = ???`.

What do you think?

Reply:
Agreed.

Why change the logging messages from headerName to HeaderName?

Reply:
+1